ClientApp/viking-vault/src/components/TopBar.tsx

```diff
@@ -1,13 +1,58 @@
 import React from 'react'
```

Comment:
Small thing, but try to group imports in a way, usually when you import an npm module you put them at the top, then group images and css files, then other components and external objects/files